src/webmon_app/reporting/tests/test_report/test_view_util.py

```diff
@@ -13,9 +13,9 @@
 import os
 import json
 from reporting import dasmon, report, reporting_app
-import httplib2
+import requests
 
-_ = [dasmon, report, reporting_app, os, httplib2]
+_ = [dasmon, report, reporting_app, os, requests]
```

Comment:
is this only to prevent linter complaints about  unused modules? can you include a code comment explaining this statement?

Reply:
I think that was the reason but looking at it I don't think you need to import them at all.